src/classes/RecipeRepository.js

```diff
@@ -40,8 +40,40 @@ class RecipeRepository {
     };
     addRecipes();
 
+    let matchingNames = this.recipeData.filter((recipe) => {
```

Comment:
Making good headway on this next method. Feels like we're almost there.

Reply:
We got this! Time to go onto Recipe!